annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastErrorReporter.java

```diff
@@ -27,6 +28,15 @@
     private static final String ANONYMOUS_EQUALS_NAMED_TEMPLATE =
         "Both anonymous and named package(s) refer to FAST package %s.";
 
+    private static final String DEFAULT_CONSTRUCTOR_ERROR_TEMPLATE =
```

Comment:
We don't need to be so specific. One error message for all cases should be enough. Something like:

Class annotated with @FastType should be instantiable with default constructor